run-tests

```diff
@@ -1,13 +1,5 @@
 #!/bin/bash
 ROOT=`dirname $0`
-OS=`uname -s`
-if [ "$OS" == "Darwin" ]; then
-  export DYLD_LIBRARY_PATH="$DYLD_LIBRARY_PATH:$ROOT/node_modules/instantclient"
-elif [ "$OS" == "Linux" ]; then
-  export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$ROOT/node_modules/instantclient"
-else
-  export PATH="$PATH:$ROOT/node_modules/instantclient"
-fi
```

Comment:
Good to skip the config steps during installation.
I can update doc https://docs.strongloop.com/display/public/LB/Installing+the+Oracle+connector for this change after this patch landed.